vertical-pod-autoscaler/pkg/admission-controller/logic/recommendation_provider.go

```diff
@@ -183,6 +128,11 @@ func (p *recommendationProvider) GetContainersResourcesForPod(pod *v1.Pod) ([]Co
 			return nil, annotations, vpaConfig.Name, err
 		}
 	}
-	containerResources := GetContainersResources(pod, *recommendedPodResources, annotations)
+	podLimitRange, err := p.limitsRangeCalculator.GetContainerLimitRangeItem(pod.Namespace)
+	// TODO: Support limit range on pod level.
```

Comment:
I just noticed this TODO is still here, is this expected?

Reply:
No, I forgot to remove it (also variable name is misleading). I'll clean up in separate PRs.

Reply:
https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/admission-controller/logic/recommendation_provider.go